### DISCUSSION TITLE
Notifications - Image

### DISCUSSION THREAD

**Discussion by timmo001** [timmo001/system-bridge#1821] [Ideas] [answered]
Add images to notifications

**Comment by timmo001** [comment, marked as answer]
https://github.com/timmo001/system-bridge/pull/1844

**Comment by Iceman248** [comment]
How big are the images shown in the notifications on Linux?

**Reply by timmo001** [reply]
The window used is 420 wide. This takes a corner of the screen up:

https://github.com/timmo001/system-bridge/blob/master/gui/systembridgegui/window/notification.py#L68

This could be increased on a per request/notification basis in the future though

**Reply by Iceman248** [reply]
That is big enough, thanks. If only I could get it to work within Home Assistant. I have only gotten it to work with the Android companion app. I posted about it here with no response in 12 days.

https://github.com/timmo001/system-bridge/discussions/1938
